engine/runtime/src/test/scala/org/enso/interpreter/test/InterpreterTest.scala

```diff
@@ -113,6 +113,7 @@ class InterpreterContext(
       .newBuilder(LanguageInfo.ID)
       .allowExperimentalOptions(true)
       .allowAllAccess(true)
+      .allowCreateThread(false)
```

Comment:
The changes in this PR remove the need for multiple thread. All `runtime` project tests can now run with single thread access only.